code/DDSCodeTester.cpp

```diff
@@ -1156,6 +1158,28 @@ class RemoteDiscoveryDomainParticipantListener : public DomainParticipantListene
 };
 //!--
 
+//!--REMOTE_TYPE_INTROSPECTION
+class TypeIntrospectionSubscriber : public DomainParticipantListener
+{
+    //!--DYNDATA_JSON_SERIALIZATION
+    void on_data_available(
+            DataReader* reader)
+    {
+        // Dynamic DataType
+        DynamicData::_ref_type new_data =
+                            DynamicDataFactory::get_instance()->create_data(dyn_type_);
+
+        std::stringstream output;
+
+        // Serialize DynamicData into JSON string format
+        json_serialize(new_data, DynamicDataJsonFormat::EPROSIMA, output);
+        std::cout << "Message received:\n" << output.str() << std::endl;
```

Comment:
I would use indentation in the example, if it's for visualization purposes it's way more readable. It may also be useful for people reading the documentation that might not know how to do so.